packages/client/src/Components/Mapping/helpers/assetsVisibleToMe.js

```diff
@@ -0,0 +1,32 @@
+import { UMPIRE_FORCE } from '../../../consts'
+
+/** provide a list of the assets that are currently visible to me
```

Comment:
Who is 'me' in this context? Rather than using words like `I`, `you`, `me` etc... it's more useful to provide real context, e.g if the `me` in the context is an umpire then the references to `me` should be replaced with `umpire`, if it's the current user then it should be replaced with `currentUser` etc...